web/frontend/src/pages/form/IndividualResult.tsx

```diff
@@ -62,82 +62,89 @@ const IndividualResult: FC<IndividualResultProps> = ({
     OUT_OF_BOUNDS = 2,
   }
 
-  const questionIcons = {
-    [RANK]: <SwitchVerticalIcon />,
-    [SELECT]: <CursorClickIcon />,
-    [TEXT]: <MenuAlt1Icon />,
-  };
-  const SubjectElementResultDisplay = (element: SubjectElement) => {
-    let titles;
-    if (isJson(element.Title)) {
-      titles = JSON.parse(element.Title);
-    }
-    if (titles === undefined) {
-      titles = { en: element.Title, fr: element.TitleFr, de: element.TitleDe };
-    }
-    return (
-      <div className="pl-4 pb-4 sm:pl-6 sm:pb-6">
-        <div className="flex flex-row">
-          <div className="align-text-middle flex mt-1 mr-2 h-5 w-5" aria-hidden="true">
-            {questionIcons[element.Type]}
+  const SubjectElementResultDisplay = useCallback(
```

Comment:
It looks like, instead of having a callback, this could/should be its own component.

Reply:
Long time no React here.

Do you mean putting `SubjectElementResultDisplay` and `displayResults` in a separate file? Or something like https://www.w3schools.com/react/react_components.asp ?

Reply:
As I was just fixing eslint warnings, I propose to leave it as-is. Same for `displayResults`.

Reply:
I meant its own file + its own exported function.
These days React components are mostly functions, no need for a class
https://react.dev/learn/your-first-component

Reply:
Here is where I don't know what to do: `displayResults` uses `SubjectElementResultDisplay`, which takes four parameters from the `IndividualResult` component. So I'm not sure how to turn my head to tell React when it should redraw, and how to write the `displayResults` and `SubjectElementResultDisplay` to make it all good.

As this was mostly a linting PR, I'll merge as-is.